core/platform/email/dev_mode_email_services.py

```diff
@@ -0,0 +1,119 @@
+# coding: utf-8
+#
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides email services api to log emails in DEV_MODE."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import logging
+import textwrap
+
+import python_utils
+
+
+def send_email_to_recipients(
+        sender_email, recipient_emails, subject,
+        plaintext_body, html_body, bcc=None, reply_to=None,
+        recipient_variables=None):
+    """Logs out sent email details to terminal console in DEV_MODE, to model
```

Comment:
"Logs out" is not really what you mean in this context, I think. Perhaps just "Logs"?